hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/handlers/CryptoGetAccountInfoHandler.java

```diff
@@ -160,8 +159,7 @@ public long estimatePendingRewards(Account account, @Nullable StakingNodeInfo st
             } else {
                 response.header(ResponseHeader.newBuilder()
                         .nodeTransactionPrecheckCode(FAIL_INVALID)
-                        .responseType(ANSWER_ONLY)
-                        .cost(0));
+                        .cost(0)); // from mono service, need to validate in the future
```

Comment:
By validate, do you mean this is a question you have? If so, you should ask it either via Slack or the modularization meeting. Or do you mean there is additional work that needs to happen? If so, create an issue for it and link the issue here. As is, this comment doesn't help me understand what needs to happen in the future

Reply:
Updated with //FUTURE: comment. I discussed this comment with @Neeharika-Sompalli.